spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -697,6 +697,7 @@ The method `BeanManager.getInterceptionFactory()` returns an `InterceptionFactor
 <T> InterceptionFactory<T> createInterceptionFactory(CreationalContext<T> ctx, Class<T> clazz);
 ----
 
+If type parameter `T` is not a class non-portable behavior results.
```

Comment:
Well the point is that how to exclude the enum so that it will result to non-portable behavior as well I guess. Stating `is an interface` is then not sufficient, but I am still not sure about better wording here.